.gitignore

```diff
@@ -1,4 +1,7 @@
 /Manifest.toml
 .DS_Store
 empty1.jl
-empty2.jl
+customaxis.jl
```

Comment:
What are these entries for?

Reply:
These are templates from which I derive the examples.